### PR TITLE
Improve exit docs and add No vulns found to output

### LINKED ISSUE #237: Results unclear when no vulnerabilities are found.
When a repository is scanned but no vulnerabilities are found, the scanner does not give you any indication that the process successfully completed. 

No results are displayed, but it isn't immediately clear if that is because no vulnerabilities were found or if something went wrong while scanning the project. 

I would expect to see a one line note saying something along the lines of "No vulnerabilities were found in your project." And perhaps have it output in green. 

I recently scanned my copy of the osv-scanner project and there were no results returned. I wasn't sure what had happened until I ran it again with json output and it returned an empty result.

### LINKED ISSUE #351: Document return code values
We have a few different cases that we currently don't document. e.g. https://github.com/google/osv-scanner/blob/898751b55d6372521911d327466a2901819942c1/cmd/osv-scanner/main.go#L149)

We should document these.

### PR DESCRIPTION
**Pull request by another-rex**
Closes #351 
Closes #237

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -171,4 +171,19 @@ osv-scanner --format json -L path/to/lockfile > /path/to/file.json
 }
 ```
 
-</details>
\ No newline at end of file
+</details>
+
+## Return Codes
```
Comment (G-Rath) [inline docs/output.md:176]: nit: I feel like this might look nicer in a table

**Review by G-Rath — APPROVED** [review]

File: docs/output.md (hunk 1 above)
Comment (another-rex) [reply docs/output.md:176]: Good idea, done!

**Comment by hayleycd** [comment]
I would need to experiment on my own branch anyway to be sure it was looking ok.

**Comment by hayleycd** [comment]
I have a working table [here](https://hayleycd.github.io/osv-scanner/output/#return-codes)

Please see #375

File: docs/output.md [hunk 2]
```diff
@@ -171,4 +171,14 @@ osv-scanner --format json -L path/to/lockfile > /path/to/file.json
 }
 ```
 
-</details>
\ No newline at end of file
+</details>
+
```
Comment (hayleycd) [inline docs/output.md:175]: Please remove changes in this file in favor of #375

**Review by hayleycd — CHANGES_REQUESTED** [review]
Remove docs. #375 has a table that I have confirmed works and the current table in this PR does not work.

File: docs/output.md (hunk 2 above)
Comment (another-rex) [reply docs/output.md]: Undid this change

**Review by hayleycd — APPROVED** [review]

**Pull request merged**